src/components/Cards/card-data/develop-cards.js

```diff
@@ -0,0 +1,46 @@
+const developCards = [
+  {
+    href: '/develop/get-started.html',
+    lightSvg: '/img/icons/buildanapp01.svg',
+    darkSvg: '/img/icons/dark/buildanapp01.svg',
+    header: 'Start building',
+    summary: 'Set up your environment and start building.'
+  },
+  {
+    href: '/develop/which-tools.html',
+    lightSvg: '/img/icons/toolguide.svg',
+    darkSvg: '/img/icons/dark/toolguide.svg',
+    header: 'Tool guide',
+    summary: 'Not sure  what to use? Check out the Tool guide.'
+  },
+  {
+    href: '/develop/terrain',
+    lightSvg: '/img/icons/terrain.svg',
+    darkSvg: '/img/icons/dark/terrain.svg',
+    header: 'Terrain',
+    summary: 'The all-in-one tool for dApp development.'
+  },
+  {
+    href: '/develop/terrain/mint-an-nft.html',
+    lightSvg: '/img/icons/mintnft.svg',
+    darkSvg: '/img/icons/dark/mintnft.svg',
+    header: 'Mint an NFT',
+    summary: 'Learn how to mint your own NFTs using Terrain.'
+  },
+  {
+    href: '/develop/localterra',
+    lightSvg: '/img/icons/localterra.svg',
+    darkSvg: '/img/icons/dark/localterra.svg',
+    header: 'LocalTerra',
+    summary: 'Run a local Terra blockchain in Docker.'
+  },
+  {
+    href: 'https://faucet.terra.money/',
+    lightSvg: '/img/icons/Faucet.svg',
+    darkSvg: '/img/icons/dark/Faucet.svg',
+    header: 'Terra Testnet Faucet',
+    summary: 'Receive test Luna to carry out transactions on Testnet.'
```

Comment:
Change to "the testnet"?  
"Testnet" is not a proper noun anywhere else and not capitalized unless it's at the beginning of a sentence.